src/components/webviews/CozyProxyWebView.js

```diff
@@ -111,6 +120,31 @@ const initHtmlContent = async ({
   })
 }
 
+/**
+ * Checks if OauthClientLimit is reached and trigger the OauthClientsLimitExceeded limit if needed
+ * Also check if the WebView rendering should be prevented and returns the result
+ *
+ * @param {CozyClient} client - CozyClient instance
+ * @param {string} slug - The application slug
+ * @param {string} href - The WebView requested href
+ * @returns true if the WebView rendering should be prevented, false otherwise
+ */
+const doesOauthClientsLimitPreventsLoading = async (client, slug, href) => {
+  const isOauthClientsLimitExeeded = await checkOauthClientsLimit(client)
+
+  if (isOauthClientsLimitExeeded) {
+    if (slug === 'home') {
+      showOauthClientsLimitExceeded(href)
+      return false
```

Comment:
Why do we want to render the home even if oauth clients limit is exceeded ?

Reply:
Here the home will be rendered behind the pop so it won't be visible by the user

We do this to improve user experience so when they fix the number of OAuth client, they will instantaneously go to the home (as it is already loaded)

More generally, we try to always have the homeView loaded for this reason (except when no user is logged)

Note that we do this for the HomeView because it has its own container. We cannot do this for cozy-app because one container is used for all cozy-app, so for example we cannot keep drive and mespapiers loaded simultaneously. But if we could we would have done it.

Reply:
Thank you for the explanation!